### DISCUSSION TITLE
服务有问题，最近突然开始疯狂访问奇奇怪怪的网站，被肉鸡？

### DISCUSSION THREAD

**Discussion by isqilinlin** [1715173329/UnblockNeteaseMusic#885] [Q&A]
今天发现服务器的内存的cpu突然异常，检查后，发现是UnblockNeteaseMusic这个服务占用异常，然后查看日志，全是这样的访问日志：
DEBUG: (server) TUNNEL
    url: "www.cosom.cn:80"
DEBUG: (server) TUNNEL
    url: "www.qfscn.org:80"
DEBUG: (server) TUNNEL
    url: "jjautopr.com:80"
DEBUG: (server) TUNNEL
    url: "jjautopr.com:80"
DEBUG: (server) TUNNEL
    url: "www.ahxuefeng.com:80"
DEBUG: (server) TUNNEL
    url: "y3333.com:80"
DEBUG: (server) TUNNEL
    url: "www.guanjinjj.com:80"
DEBUG: (server) TUNNEL
    url: "bpmrzxd.com:80"
DEBUG: (server) TUNNEL
    url: "www.cosom.cn:80"
DEBUG: (server) TUNNEL
    url: "www.guanjinjj.com:80"
DEBUG: (server) TUNNEL
    url: "www.qfscn.org:80"
DEBUG: (server) TUNNEL
    url: "www.yangletou.com:80"
DEBUG: (server) TUNNEL
    url: "www.qfscn.org:80"
DEBUG: (server) TUNNEL
    url: "bpmrzxd.com:80"
    
    一直在高频执行访问，无语了，显然是被肉鸡了。

**Comment by 1715173329** [comment]
unm 本质只是一个 HTTP 代理。
请参考文档启用严格模式并指定 endpoint。
